inc/saihostintf.h

```diff
@@ -526,6 +526,10 @@ typedef enum _sai_hostif_attr_t
 
     /** Set the operational status for this host interface [bool] (default to false) */
     SAI_HOSTIF_ATTR_OPER_STATUS,
+    
+    /** Set the cos queue number to be used for packets going out through this interface [uint32_t] 
+    * (default 0) This is an index. */
+    SAI_HOSTIF_ATTR_COS_QUEUE,
```

Comment:
Will this mean the same cos queue is used for all packets? There is a new attribute SAI_PACKET_EGRESS_QUEUE proposed in the Host IF enhancements proposal (SAI-Proposal-4-host-if-ver7.docx) which would solve this requirement?